### PR TITLE
Created model and migration for posts

### PR DESCRIPTION
**Pull request by stefanmancevski**
Created a model + migration to allow for post creation

### CODE REVIEW

**Comment by jacoblipson** [comment]
Approved

File: db/migrate/20190509204900_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CreatePosts < ActiveRecord::Migration[5.1]
+  def change
+    create_table :posts do |t|
+      t.string :content
+      t.references :user
```
Comment (phantomphildius) [inline db/migrate/20190509204900_create_posts.rb:5]: probably should have added `foreign_key: true, null: false` to ensure faster look ups and no orphaned posts @stefanmancevski !

Comment (phantomphildius) [reply db/migrate/20190509204900_create_posts.rb:5]: my mistake

Comment (stefanmancevski) [reply db/migrate/20190509204900_create_posts.rb:5]: ¯\_(ツ)_/¯

**Pull request merged**